### ISSUE TITLE
Looting and Quest Items

### ISSUE THREAD

**Issue by Aldavor** [Ludovicus/WoW-Pro-Guides#419]
At this stage, just an observation... since Blizzard changed the looting system so that certain quest items no longer appear in your inventory/bag, the "l" (loot) step doesn't auto-check off/complete once they are obtained. Is this something that can be checked within the main .lua code, or does the method need to be revised? I'm currently changing the 'l' steps to using 'C' but it doesn't always work, and could end up being a real pain to go through and replace all such references...

**Comment by Ludovicus** [comment]
Blizzard only started using the "virtual loot items" with WoD.  We were worried that everything was going to change, but when we played through a couple of random zones in Beta, it looked like nothing had changed in the old world.

We suspect that as Blizzard revamps some of the older zones that quests will be updated and we will need to change from using |L|xxx| to |QO|#|.   

We could check to see if the L item exists in the game and is a quest item, but if it does not, then we have no clue how to tie it to the current quest if there is more than one QO.  If the quest has exactly one QO, then we could substitute that one.    That is the only thing we could do in the code.  It is risky, as item queries trigger a call back to the server and need to be repeated later if they return no data, which complicates matters.

**Comment by Aldavor** [comment]
If the lootable object is an actual quest objective (as in sub-element of the quest) QO|#| can definitely be used - I've replaced as such in the guides I've updated and all seems to work fine. I think it may only be an issue with non-quest related looted objects. Regarding several QOs, can they not be put onto individual C lines, perhaps with a sticky to indicate an As You Go type sequence?

**Comment by Ludovicus** [comment]
Hmm, Now you have me wondering if I could *auto-generate* a set of C steps from with the QO lines from the quest itself if I run into a C step sans a QO or L tag.  The problem is that I would not know what to use for the M tag for each of the QO.  The would likely be different and sometimes the order of the QO's that is good for questing is not the listing order.

**Comment by Ludovicus** [comment]
OK.   I have made enough QO corrections to quests that I think we need to convert them all.
It should be simple enough to scrape the necessary information from WoWHead and convert the text in a QO into an index.   

Any l steps with L tags are more difficult to adapt. It may be possible to map back the item # to a quest and then change it into a QO.

But blizzard is definitely revamping the older zones now.   I ran into plenty of QO problems in Silverpine Forest, for example.
